libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1138,6 +1138,25 @@ string YulUtilFunctions::negateNumberCheckedFunction(Type const& _type)
 		});
 }
 
+string YulUtilFunctions::zeroValueFunction(Type const& _type)
+{
+	solUnimplementedAssert(_type.sizeOnStack() == 1, "Stacksize not yet implemented!");
+	solUnimplementedAssert(_type.isValueType(), "Delete for non-value types not yet implemented");
+
+	string const functionName = "zero_value_for_" + _type.identifier();
+
+	return m_functionCollector->createFunction(functionName, [&]() {
+		return Whiskers(R"(
+			function <functionName>() -> ret {
+				<body>
```

Comment:
```suggestion
				ret := 0
```

Reply:
i am expecting that we're adding more cases here soon though

Reply:
Ok!